x-pack/filebeat/module/suricata/eve/ingest/pipeline.json

```diff
@@ -254,9 +256,9 @@
             }
         },
         {
-            "set": {
-                "field": "event.kind",
-                "value": "event"
+            "script": {
+                "lang": "painless",
+                "source": "def t = ctx.suricata?.eve?.event_type; if (t == \"stats\") {\n  ctx['event']['kind'] = \"metric\";\n} else if (t == \"alert\") {\n  ctx['event']['kind'] = \"alert\";\n  ctx['event']['category'] = \"network_traffic\";\n} else {\n  ctx['event']['kind'] = \"event\";\n  ctx['event']['category'] = \"network_traffic\";\n}"
             }
```

Comment:
For the reviewers sake, this is what the code looked like before I made it JSON.
```
  - script:
      lang: painless
      source: >-
        def t = ctx.suricata?.eve?.event_type;
        if (t == "stats") {
          ctx['event']['kind'] = "metric";
        } else if (t == "alert") {
          ctx['event']['kind'] = "alert";
          ctx['event']['category'] = "network_traffic";
        } else {
          ctx['event']['kind'] = "event";
          ctx['event']['category'] = "network_traffic";
        }
```